searxng/settings.yml

```diff
@@ -8,4 +8,4 @@ server:
 ui:
   static_use_hash: true
 redis:
-  url: redis://redis:6379/0
+  url: redis://valkey:6379/0
```

Comment:
This doesn't add any value to switch to a service name called "valkey". Please revert to the service name "redis" like explained in https://github.com/searxng/searxng-docker/pull/230#discussion_r1585199869